src/native.js

```diff
@@ -229,9 +364,31 @@ export function getAllAssetsMessage(data, adObject) {
   const message = {
     message: 'assetResponse',
     adId: data.adId,
-    assets: []
   };
 
+  // Provide a mapping from legacy asset to openRTB id, so legacy creatives can still display ads
```

Comment:
Could this be done by providing assets in the legacy format directly? If it can be done in a way that's backwards compatible with current PUC behavior, it should make rollout significantly easier.